Package.swift

```diff
@@ -261,6 +263,15 @@ var package = Package(
       ]
     ),
 
+    .target(
+      name: "PointFreeDependencies",
+      dependencies: [
+        "Models",
+        "PointFreeRouter",
+        .product(name: "Dependencies", package: "swift-dependencies"),
+      ]
+    ),
```

Comment:
I'm putting all the dependency keys in here for now, but we could split into more modules later if we want. Unfortunately we have to depend on `PointFreeRouter`, which is pretty heavy, because the site's routes _and_ router all live in there. We should probably break the routes out into its own module.